bluebrain/repo-bluebrain/packages/neurodamus/package.py

```diff
@@ -0,0 +1,428 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+import shutil
+from contextlib import contextmanager
+
+from llnl.util import tty
+
+from spack.package import *
+
+# Definitions
+_CORENRN_MODLIST_FNAME = "coreneuron_modlist.txt"
+_BUILD_NEURODAMUS_FNAME = "build_neurodamus.sh"
+
+
+class Neurodamus(Package):
+    """The next-generation AllInOne Neurodamus deployment.
+
+    Neurodamus includes BBP's simulation suite with all internal sim models.
+    """
+
+    homepage = "https://bbpgitlab.epfl.ch/hpc/sim/neurodamus-models"
+    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-models.git"
+
+    version("develop", submodules=True)
+    # Let the version scheme be different to avoid mixing with old neurodamus-**?
+    version("2023.05", tag="0.0.2", submodules=True)
+
+    variant("synapsetool", default=True, description="Enable SynapseTool reader (for edges)")
+    variant("mvdtool", default=True, description="Enable MVDTool reader (for nodes)")
+    variant("coreneuron", default=False, description="Enable CoreNEURON Support")
```

Comment:
this should be removed now (or true) i.e. coreneuron support should be enabled by default.

Reply:
Remove, please.